src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -464,11 +474,25 @@ private void the_txout_is_returned()
             txOutResponse.Value.Should().Be(this.transferAmount);
         }
 
+        private void staking_information_is_returned()
+        {
+            var stakingInfoModel = JsonDataSerializer.Instance.Deserialize<GetStakingInfoModel>(this.responseText);
+            stakingInfoModel.Enabled.Should().Be(false);
+            stakingInfoModel.Staking.Should().Be(false);
+        }
+
+        private void a_bad_request_error_is_returned()
+        {
+            this.response.StatusCode.Should().Be(StatusCodes.Status400BadRequest);
```

Comment:
I think 405 might be more appropriate....

```
405 Method Not Allowed
A request method is not supported for the requested resource; for example, a GET request on a form that requires data to be presented via POST, or a PUT request on a read-only resource.
```

Reply:
Agree, updated.